holoviews/plotting/renderer.py

```diff
@@ -257,3 +284,10 @@ def state(cls):
         """
         yield
 
+
+    @classmethod
+    def validate(cls, options):
+        """
+        Validates a dictionary of options set on the backend.
```

Comment:
I think the docstring should be something like ``"Validate an options dictionary for the renderer"'. The reason is that I would say that 'renderer' is a holoviews term (subclass of ``Renderer``) whereas the 'backend' is the library or 3rd party tool used by the renderer. Very closely related, I agree but not quite the same thing.